maintenance/passport-azure-ad/package.json

```
@@ -46,7 +49,8 @@
     "valid-url": "^1.0.6"
   },
   "scripts": {
-    "test": "grunt run_tests"
+    "test": "grunt run_tests",
+    "lint": "./node_modules/.bin/eslint lib/**"
```

I think this can just be

```suggestion
    "lint": "eslint lib/**"
```

---

After discussion, leaving the lint command as is. Changing to `eslint lib/**` results in `Invalid ecmaVersion` error.